### ISSUE TITLE
[DOC] RCA should maybe be in "Supervised algorithms"

### ISSUE THREAD

**Issue by wdevazelhes** [metric-learn/metric-learn#237]
Currently `RCA` is in Pairs Learners in the doc, where it shouldn't. It should probably be in "Supervised"

**Comment by bellet** [comment]
I agree it shouldn't be in pairs learners. It is not supervised either though (only need to know that groups of points share the same label, not the label itself). We could put it there if we do not want to create a new section in the doc for learning from chunklets (which will contain only RCA).

**Comment by bellet** [comment]
Note also that we are currently inconsistent as RCA appears under Supervised Learning Algorithms in the Package Contents section f the doc:
http://contrib.scikit-learn.org/metric-learn/metric_learn.html#supervised-learning-algorithms
